### PR TITLE
Closes #295 add PR website [create website]

### LINKED ISSUE #295: Add workflow to create PR website

### PR DESCRIPTION
**Pull request by manciniedoardo**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update).
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files.
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions]
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately.
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralophtha (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralophtha site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiralophtha/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`.
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`.
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by bundfussr** [comment]
I think you need to add `DT` and `htmltools` to `Suggests:`. I'm not sure why it worked before.

File: NEWS.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ and provided by [kapa.ai](https://docs.kapa.ai/kapa-for-open-source). (#2887)
 
 * Updated `{lintr}` configurations to use central configurations from `{admiraldev}`. (#280)
 
+* Updated `.github/common.yml` to add new PR website workflow, accessible by placing the keyword "[create website]" in PR titles. (#295)
```
Comment (bundfussr) [inline NEWS.md]: I assume this should be in a new section for the development version.

File: DESCRIPTION
Comment (bundfussr) [inline DESCRIPTION:1]: I think the version number should be updated.

Comment (manciniedoardo) [reply DESCRIPTION:1]: thanks, forgot that there was now new section - now done

File: NEWS.md (hunk 1 above)
Comment (manciniedoardo) [reply NEWS.md]: thanks, forgot that there was now new section - now done

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+## Updates to Templates
+
+## Updates to Documentation
+
+## Various
+
+<details>
+<summary>Developer Notes</summary>
+
+* Updated `{lintr}` configurations to use central configurations from `{admiraldev}`. (#280)
```
Comment (bundfussr) [inline NEWS.md]: This should be removed because it is from 1.4.0.

File: NEWS.md (hunk 1 above)
Comment (bundfussr) [reply NEWS.md]: I think the item should be moved to the new section.

Comment (bundfussr) [reply NEWS.md]: Now it is in the new section but still here.

**Review by bundfussr — COMMENTED** [review]
I think you can remove `.github/CODEOWNERS` because it refers to Dinakar.

**Review by bundfussr — APPROVED** [review]
